asrtoolkit/alignment/preprocess_txt.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+import os
+
+from asrtoolkit.clean_formatting import clean_up
+
+
+def parse_transcript(transcript,
+                     speaker="unknown",
+                     gender="male",
+                     token_idx_offset=0):
+    """
+    Given reference transcript (and ideally speaker and gender),
+    Return a list of tokenized word objects
+    """
+    if os.path.exists(transcript):
```

Comment:
It may be good to have an error messge if the `transcript` doesn't exist

Reply:
clarifying expectations